Schemas/TEILex0/TEILex0.examples/examples.xml

```diff
@@ -835,114 +837,335 @@
             <!--... -->
          </entry>
          <!--arabic-->
-         <entry type="wordFamily" xml:id="hdjs63" xml:lang="ar">
-            <form type="root"><orth>ﺍﺮﻛ</orth></form>
+
+         <entry type="wordFamily" xml:id="ark" xml:lang="ar">
+            <form type="root">
+               <orth>ارك</orth>
+            </form>
             <pc>:</pc>
-            <entry n="١-" type="wordForm" xml:id="xhd32" xml:lang="ar">
-               <form type="lemma">
-                  <orth>ﺃَﺮَﻛَ</orth> -ُ ﺃُﺮُﻛًﺍ</form>
-               <sense> ﺎﻠﺠﻤﻟُ: ﺎﻜﻟ ﻭﺮﻗ ﺍﻷﺭَﺎﻛِ ٠ ﺃَﺮَﻛَ ٠- ﺃَﺮَﻛًﺍ ﺎﻠﺠﻤﻟُ ﺎﺸﺘﻛﻯ ﺐﻄﻨﻫُ ﻢﻧ ﺄَﻜﻟ
-                  ﺍﻷﺭَﺎﻛِ ٠ ﺍﻷﺭﺎﻛ (ﻥ): ﻭﺎﺣﺪﺘﻫُ ﺃَﺭﺎﻛﺓ ﺝ ﺃُﺮُﻛ ﻭ ﺃَﺭَﺎﺌِﻛ. ﺶﺟﺭ ﺫﻭ ﺵﻮﻛ. ﻁﻮﻴﻟ ﺎﻠﺳﺎﻗ،
-                  ﻚﺜﻳﺭ ﺎﻟﻭﺮﻗ ﻭ ﺍﻸﻐﺻﺎﻧ. ﺥﻭَّﺍﺭ ﺎﻠﻋﻭﺩ ﺖﺘَّﺧﺫ ﻢﻨﻫ ﺎﻠﻤﺳﺍﻮﻴﻛ</sense>
+            <entry n="1" type="wordFamily" xml:id="ark1" xml:lang="ar">
+               <num>١-</num>
+               <entry type="mainEntry" xml:id="ark1_1" xml:lang="ar">
+                  <form type="lemma">
+                     <orth>ﺃَﺮَﻛَ</orth>
+                  </form>
+                  <form type="scheme">
+                     <orth>ُ- </orth>
+                  </form>
+                  <form type="inflected">
+                     <gramGrp>
+                        <gram type="case" resp="khemakhem" value="accusative"/>
+                     </gramGrp>
+                     <orth>أُُُرُوكًا</orth>
+                  </form>
+                  <sense xml:id="ark1_1_sense1">
+                     <cit type="example">
+                        <quote>الجملُ</quote>
+                     </cit><pc>:</pc> 
+                     <def>اكل ورق الأَرَاك</def>
+                     ِِ</sense>
```

Comment:
@MedKhem can you check this —  you have a kasra (according to my Unicode utility, I am not trying to pretend I know any Arabic letters :) outside the def. we can't have text content inside sense.

Reply:
well seen +1 . I fixed that in my latest commit